include/linux/tee_drv.h

```diff
@@ -192,6 +196,7 @@ struct tee_shm {
 	struct tee_device *teedev;
 	struct tee_context *ctx;
 	struct list_head link;
+	struct list_head ocall_link;
```

Comment:
I'd prefer if we didn't need this.

Reply:
Come to think of it, me too. This is an issue if the same SHM is used in multiple OCALLs. It's not possible now, but it may be in the future.